openedx/core/djangoapps/content_tagging/rules.py

```diff
@@ -282,6 +282,7 @@ def can_change_taxonomy_tag(user: UserType, tag: oel_tagging.Tag | None = None)
 rules.set_perm("oel_tagging.delete_taxonomy", can_change_taxonomy)
 rules.set_perm("oel_tagging.view_taxonomy", can_view_taxonomy)
 rules.set_perm("oel_tagging.export_taxonomy", can_view_taxonomy)
+rules.set_perm("oel_tagging.import_taxonomy", can_create_taxonomy)
```

Comment:
These can be removed too, since we're removing them from oel_tagging.

Reply:
Done!